server/repository/src/migrations/v2_01_00/pg_enums.rs

```diff
@@ -37,5 +37,12 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
         "#,
     )?;
 
+    sql!(
+        connection,
+        r#"
+            ALTER TYPE permission_type ADD VALUE 'CENTRAL_SERVER_ADMIN';
```

Comment:
There is already ServerAdmin or is this one different? if yes a comment would be good.